exercises/sum.cpp

```diff
@@ -6,3 +6,18 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+int main(){
+  int a;
+  int b;
+  cout << "Enter first number: ";
+  cin >> a;
+  cout << "and second number: ";
+  cin >> b;
+  int sum = a+b;
+  cout << "The sum is: " <<  sum << "\n";
```

Comment:
C'è da aggiungere, alla fine del **main**, il 
```c
return 0;
```

Oppure, ancora meglio, si potrebbe aggiungere in cima al file
```c
#include <iostream>
#define EXIT_SUCCESS 0

using namespace std
```
e aggiungere alla fine del **main**

```c
return EXIT_SUCCESS;
```